apps/scan/backend/schema.sql

```diff
@@ -9,8 +9,6 @@ create table election (
   ballot_count_when_ballot_bag_last_replaced integer not null default 0,
   is_sound_muted boolean not null default false,
   is_ultrasonic_disabled boolean not null default false,
-  cvrs_backed_up_at datetime,
-  scanner_backed_up_at datetime,
```

Comment:
Not only can we delete logic for the old-style of backup; we can also delete logic for the check that prevents unconfiguring a machine until you've backed. Continuous export makes it such that you're essentially always backed up